homeassistant/components/media_player/universal.py

```diff
@@ -357,6 +357,12 @@ def source_list(self):
         return self._override_or_child_attr(ATTR_INPUT_SOURCE_LIST)
 
     @property
+    def shuffle(self):
+        """Boolean if shuffling is enabled."""
+        return self._override_or_child_attr(ATTR_MEDIA_SHUFFLING) \
+            in [True, STATE_ON]
```

Comment:
This should always be just comparing against `True`. So just return the result directly.